### PR TITLE
Milestone 4: Set Up Controllers

### PR DESCRIPTION
**Pull request by topeogunleye**
## Milestone 4: Set Up Controllers

In this Pull Request, I:

- Added Routes for posts, users.
- Added PostsController and UsersController
- Added Post and User Model
- Added template for PostsController#index
- Added template for UsersController#index
- Added template for PostsController#show
- Added template for UsersController#show

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users, only: [:show, :index] 
+  resources :posts, only: [:show, :index]
```
Comment (alexander16108) [inline config/routes.rb]: your statement requires a `do` at the end of this block of code eg 
`
resources :users, only: [:show, :index] do
resources :posts, only: [:show, :index] 
`

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @topeogunleye , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (GzimAsani) [inline config/routes.rb]: - [ ] Kindly implement the requirement asked by the previous reviewer, check the picture :arrow_down:  as an example
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @topeogunleye            :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the controllers :+1: 
 
 ## Required Changes 

- [ ] Check comment below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (GzimAsani) [reply config/routes.rb]: You made the change after you submitted the ask for code review, code looks good but please remember to always ask for code review after you have done the change

Comment (topeogunleye) [reply config/routes.rb]: Thank you @GzimAsani

**Review by ShadyShawkat — APPROVED** [review]
## Hi @topeogunleye, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — COMMENTED** [review]
Hello @topeogunleye  👋,

_This is an additional review as a result of the second opinion request._

Please correct me if I am wrong, but you wrongly requested an SO using this PR link.
Until confirmation, this SO will not be considered.

Keep rocking 🚀!

**Pull request merged**